### PR TITLE
revise explain format by list

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What is changed, added or deleted? (Required)

revise explain format by list

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://asktug.com/t/topic/903234

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@Yisaer 对于 Line 40 的 operator info，有位用户有这样的提议：https://asktug.com/t/topic/903234/4。

File: sql-statements/sql-statement-explain.md [hunk 1]
```diff
@@ -185,9 +185,15 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 
 在 `EXPLAIN` 中使用 `FORMAT = "xxx"` 语法可以指定输出的内容和格式。
 
-如果 `EXPLAIN` 语句中未指定 `FORMAT`，或指定 `FORMAT = "row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md)。
+| FORMAT | 作用 |
+| ------ | ------ |
+| 未指定  | 同 row |
+| row    | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
+| brief  | `EXPLAIN` 语句输出结果中的算子 ID 将被隐藏，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化 |
```
Comment (time-and-fate) [inline sql-statements/sql-statement-explain.md:192]: 稍微有点语病。
按照这篇文档上面部分的说法，`Projection_3` 这个整体才叫 ID，而不是单指那个数字，所以之前的文档是写的“简化”而不是“隐藏”。

File: sql-statements/sql-statement-explain.md [hunk 2]
```diff
@@ -185,9 +185,15 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
+| dot    | `EXPLAIN` 语句将输出 dot 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
 
-如果在 `EXPLAIN` 中指定了 `FORMAT = "brief"`，那么 `EXPLAIN` 语句输出结果中的算子 ID，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化：
+
+在 `EXPLAIN` 中指定 `FORMAT = "brief"` 时，示例如下:
```
Comment (time-and-fate) [inline sql-statements/sql-statement-explain.md:196]: 下面介绍 `dot` 部分的措辞也可以对应地稍微简化一下，现在和上面稍微有点重复。

**Review by time-and-fate — APPROVED** [review]
提了两个小措辞上的小问题，技术上没啥问题，先 approve 了。

**Comment by Yisaer** [comment]
> @Yisaer 对于 operator info，有位用户有这样的提议：https://asktug.com/t/topic/903234/4 。 image

这是对 explain 功能的建议，和这个 issue 无关吧

File: sql-statements/sql-statement-explain.md [hunk 3]
```diff
@@ -185,9 +185,15 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 
 在 `EXPLAIN` 中使用 `FORMAT = "xxx"` 语法可以指定输出的内容和格式。
 
-如果 `EXPLAIN` 语句中未指定 `FORMAT`，或指定 `FORMAT = "row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md)。
+| FORMAT | 作用 |
+| ------ | ------ |
+| 未指定  | 同 row |
+| row    | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
+| brief  | `EXPLAIN` 语句输出结果中的算子 ID 将被简化，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化 |
+| dot    | `EXPLAIN` 语句将输出 dot 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
```
Comment (shichun-0415) [inline sql-statements/sql-statement-explain.md:193]:
```suggestion
| `row`    | `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md) |
| `brief`  | `EXPLAIN` 语句输出结果中的算子 ID 将被简化，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化 |
| `dot`    | `EXPLAIN` 语句将输出 dot 格式的执行计划，可以通过 `dot` 程序（在 `graphviz` 包中）生成 PNG 文件 |
```

**Pull request merged**